packages/about/spec/update-manager-spec.js

```diff
@@ -1,5 +1,10 @@
 const UpdateManager = require('../lib/update-manager');
 
+const REPO_OWNER = process.env.REPO_OWNER || 'atom';
+const MAIN_REPO = process.env.MAIN_REPO || 'atom';
+const NIGHTLY_RELEASE_REPO =
+  process.env.NIGHTLY_RELEASE_REPO || 'atom-nightly-releases';
```

Comment:
Same as above.

Reply:
At least this is a spec file (test definition). I think the tests are only run in CI, so this one might work.

Reply:
Actually, in CI, with both files being updated like this, it should _run the tests_ against our fork's GitHub releases page. ✔️ 

That said, you are right that this would not help end-users. I agree we need to hard-code the fork's URL, or otherwise make it available at runtime.